salt/_modules/caasp_orch.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+
+
+def __virtual__():
+    return "caasp_orch"
+
+
+def sync_all():
+    '''
+    Syncronize everything before starting a new orchestration
+    '''
+    __utils__['caasp_log.debug']('orch: synchronizing the mine')
```

Comment:
we can refactor this 2 lines  in a `def sync_mine()` function

Reply:
and calling it in the `all`

Reply:
I wouldn't refactor so early this code. If we need to have a `sync_mine()` at some moment we can refactor out this code and do as you suggest, but until then I think the only case we have is a `sync_all()`.